assembl/fabfile.py

```diff
@@ -3296,45 +3304,56 @@ def secure_sshd_fail2ban():
 
 
 @task
-def ensure_private_configs():
+def migrate_important_information(path, to=None):
     """
-    Private configs are created once for a server. During CI/CD, we want these configurations
-    to survive each version upgrade
+    Move important files that are meant to survive moving to an automated deployment strategy
     """
-    remote_random_path = os.path.join(env.projectpath, 'random.ini')
-    private_config_path = normpath(os.path. join(env.projectpath, '..', 'global_configs'))
-    if exists(remote_random_path):
-        run('mkdir -p %s' % private_config_path)
-
+    if exists(path):
         # Ensure that the server's random.ini survives (under migration). New servers will generate own random.ini
-        remote_random_path_destination = normpath(os.path.join(private_config_path, 'random.ini'))
+        basename = os.path.basename(path)
+        remote_random_path_destination = normpath(to, basename)
         if not exists(remote_random_path_destination):
-            run('cp %s %s' % (remote_random_path, remote_random_path_destination))
+            run('cp -r %s %s' % (path, remote_random_path_destination))
```

Comment:
If, by any chance, the random path already exists, this command won't fail and you have no clue that the copy command wasn't executed.